src/push/bundler.ts

```diff
@@ -51,7 +51,7 @@ export class Bundler {
         write: false,
         minifyWhitespace: true,
         sourcemap: 'inline',
-        external: ['@elastic/synthetics'],
+        external: ['@elastic/synthetics', 'playwright-core'],
```

Comment:
We could do this, but we would end up creating broken experience for the users if they decided to pull some code from PW. 

Allowing this is better for the UX side than having a broken experience. WDYT?

Reply:
Better way would be to disallow it as per your PR and have a symbolic link in the Heartbeat side when we set up project monitors so we dont have a broken experience for users.